components/omega/doc/devGuide/QuickStart.md

```diff
@@ -2,7 +2,196 @@
 
 # Quick Start for Developers
 
-:::{admonition} Quick Start for Developers is coming soon
-OMEGA is still in such an early stages of development that we don't even have
-a quick start for developers yet.
-:::
+
+## Check out the code
+
+Make sure you have added the SSH key for the machine you are using to your
+GitHub account, see
+[Adding a new SSH key to your GitHub account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account).
+
+Clone the develop branch of the Omega repo:
+```sh
+git clone git@github.com:E3SM-Project/Omega.git develop
+```
+
+Rename `origin` to `E3SM-Project/Omega` for clarity:
+```sh
+git remote rename origin E3SM-Project/Omega
+```
+
+Consider adding some other remotes, e.g.:
```

Comment:
It might be good to add a section about making a fork for completeness (or just mention it somewhere around here). We should probably convey somewhere that PRs should be submitted for branches on developer forks.

Reply:
I added this above.